procedures/auxtel/observation_procedures/Parabolic_Focus_Determination_Data_Acq.ipynb

```diff
@@ -9,23 +9,42 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    index = -int((test_date) + str(np.random.randint(10, 99)))
</pre><p>oh this is pretty dangerous. It can create a negative index which isn't allowed (and currently isn't handled well!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Fixed already in your commit</p>